src/roulette.py

```diff
@@ -126,3 +126,172 @@ def get(self, bin: int) -> Bin:
+                                 street_bet_odds)
+            wheel.addOutcome(number, street_bet)
+            wheel.addOutcome(number + 1, street_bet)
+            wheel.addOutcome(number + 2, street_bet)
+
+    @staticmethod
+    def create_corner_bets(wheel: Wheel) -> None:
+        odds_for_corner_bet = 8
+
+        corner_bet_numbers = {number * 3 + 1 for number in range(11)} | {number * 3 + 2 for
+                                                                         number in range(11)}
+        for number in corner_bet_numbers:
+            first_column_corner_bet = Outcome(str(number) + "-" + str(number + 1) + "-" +
+                                              str(number + 3) + str(number + 4),
+                                              odds_for_corner_bet)
+            wheel.addOutcome(number, first_column_corner_bet)
+            wheel.addOutcome(number + 1, first_column_corner_bet)
+            wheel.addOutcome(number + 3, first_column_corner_bet)
+            wheel.addOutcome(number + 4, first_column_corner_bet)
+
+    @staticmethod
+    def create_line_bets(wheel: Wheel) -> None:
+        line_bet_odds = 5
+        line_bet_numbers = {number * 3 + 1 for number in range(11)}
+        for number in line_bet_numbers:
+            line_bet = Outcome(str(number) + "-" + str(number + 1) + "-" + str(number + 2) + "-" +
+                               str(number + 3) + "-" + str(number + 4) + "-" + str(number + 5),
+                               line_bet_odds)
+            wheel.addOutcome(number, line_bet)
+            wheel.addOutcome(number + 1, line_bet)
+            wheel.addOutcome(number + 2, line_bet)
+            wheel.addOutcome(number + 3, line_bet)
+            wheel.addOutcome(number + 4, line_bet)
+            wheel.addOutcome(number + 5, line_bet)
+
+    @staticmethod
+    def generate_five_bets(wheel: Wheel) -> None:
+        odds_for_five_bet = 6
+        five_bet_name = "00-0-1-2-3"
+        five_bet = Outcome(five_bet_name, odds_for_five_bet)
+        bin_indexes_for_five_bet = {0, 1, 2, 3, 37}
+
+        for index in bin_indexes_for_five_bet:
+            wheel.addOutcome(index, five_bet)
+
+    @staticmethod
+    def create_even_money_bets(wheel: Wheel) -> None:
+        even_bet_odds = 1
+        red_bet_name = "Red"
+        black_bet_name = "Black"
+        low_bet_name = "Low"
+        high_bet_name = "High"
+        even_bet_name = "Even"
+        odd_bet_name = "Odd"
+
+        even_bet_numbers = {_ for _ in range(1, 37)}
+        red_bet_numbers = {1, 3, 5, 7, 9, 12, 14, 16, 18, 19, 21, 23, 25, 27, 30, 32, 34, 36}
+
+        red_bet = Outcome(red_bet_name, even_bet_odds)
+        black_bet = Outcome(black_bet_name, even_bet_odds)
+        low_bet = Outcome(low_bet_name, even_bet_odds)
+        high_bet = Outcome(high_bet_name, even_bet_odds)
+        even_bet = Outcome(even_bet_name, even_bet_odds)
+        odd_bet = Outcome(odd_bet_name, even_bet_odds)
```

Comment:
pylint complains about "Too many local variables (17/15)".  what about creating Outcome in place while invoking `addOutcome`